charts/kargo/values.yaml

```diff
@@ -3,6 +3,13 @@
 ## This is a YAML-formatted file.
 ## Declare variables to be passed into your templates.
 
+## @section Global Parameters
+global:
+  ## @param global.env env to pass to all deployed Deployments and CronJobs
```

Comment:
```suggestion
  ## @param global.env Supplemental environment variables for all Kargo workloads.
```

Reply:
I was also wondering why no `globa.envFrom`?

Reply:
Just because I stuck to the argoCD helm template where there was no global.envFrom. We can add it if you want to.